next/utils/forms.ts

```diff
@@ -403,60 +471,88 @@ export const useFormStepper = (eformSlug: string, schema: RJSFSchema) => {
   }
 
   const setUniqueErrors = (newErrors: RJSFValidationError[], actualStepIndex: number) => {
-    const updatedErrors: RJSFValidationError[][] = []
-    const stepsRange = [...Array.from({ length: stepIndex + 1 }).keys()]
-
-    stepsRange.forEach((id) => {
-      const stepErrors = errors[id]
-      if (id === actualStepIndex) {
-        updatedErrors.push([...newErrors])
-      } else if (stepErrors) {
-        updatedErrors.push([...stepErrors])
-      } else {
-        updatedErrors.push([])
-      }
-    })
+    // update form errors - update even if there is no error
+    const actualStepKey = stepData[actualStepIndex].stepKey
```

Comment:
actual doesn't mean "aktuálny" - if that's the implied meaning then

```suggestion
    const currentStepKey = stepData[actualStepIndex].stepKey
```